### DISCUSSION TITLE
File input snippet not supporting locales

### DISCUSSION THREAD

**Discussion by fcanacmarquis** [twbs/bootstrap#40751] [Ideas]
Hi,

I would love the File input label showing "Select File(s)" to support multiple locales such as french. So both the button label and the file count would show in the correct locale.

[https://getbootstrap.com/docs/5.3/forms/form-control/#file-input](https://getbootstrap.com/docs/5.3/forms/form-control/#file-input)

**Comment by louismaximepiton** [comment]
Hey,

I think the Bootstrap file input uses the default browser text and so is adapted to the local language directly. For instance, I got this:
![image](https://github.com/user-attachments/assets/7af3949b-7ebe-4671-9015-d0b30b9dc478)

Instead of:
![image](https://github.com/user-attachments/assets/08677882-900f-458b-92a8-0b1ea54675b4)
